libs/filamat/src/shaders/CodeGenerator.cpp

```diff
@@ -64,6 +64,7 @@ io::sstream& CodeGenerator::generateProlog(io::sstream& out, ShaderType type,
                 out << "#version 450 core\n\n";
             } else {
                 out << "#version 410 core\n\n";
+                out << "#extension GL_ARB_shading_language_packing : enable\n\n";
```

Comment:
Are we guaranteed to have this extension?

Reply:
No but it doesn't mater right? When you specify `enable` vs. `require`. I'm checking at runtime and injecting the appropriate code if it's not there. The spec says that if the extension is not supported, it's just a warning.

Reply:
> Confirmed that this fixes the issue on my machine, although maybe the #extension line needs to be conditional.

@prideout did you check with a debug or release build?

Reply:
The bug repro and the verification of the fix were both using a release build of Filament.  (I'm actually unable to build a debug build of Filament on my Windows machine because matc crashes during the build, I haven't looked into that yet.)

Reply:
Nice, I think if it works with a release build, we're good.